README.md

```diff
@@ -33,7 +33,7 @@ After reboot, a firmware partition could become committed by calling
 reboots back into the old firmware after the `commit_timeout` seconds.
 Example usage:
 
-<pre class="command-line language-bash" data-user="chris" data-host="localhost" data-output="2-100"><code>mos call OTA.Update '{"url": "http://1.2.3.4/fw.zip", "commit_timeout": 300}'</code></pre>
+<pre class="command-line language-bash" data-user="chris" data-host="localhost" data-output="2-100"><code>mos call OTA.Update '{"url": "http://192.0.2.4/fw.zip", "commit_timeout": 300}'</code></pre>
```

Comment:
keep it as 1.2.3.4 - it's obvious it's a fake address, whereas 192.0.2.4 looks magic

Reply:
It's not a fake address, it's assigned. You could change it to example.com.

Reply:
i thought so but not everyone knows that. address in the command is a placeholder and it should _look_ like one. 1.2.3.4 looks like a placeholder, 192.0.2.4 does not.

Reply:
To be honest, I don't really care all that much :)  Like I said, I was writing some code to help correct the links in the mongoose docs because all the dead links annoyed me. I'd be reluctant to have clearly incorrect information in docs and wanted to point it out to you. I do see your point, though. Maybe "example.com" or "<insert the url of your server here>" would be alternatives, but ultimately it's your decision/responsibility.

Reply:
to sum up : foo.com -> example.com change LGTM, but please keep 1.2.3.4 here.

Reply:
feel free to make any changes you see fit! Thanks!